src/main/java/appeng/core/api/definitions/ApiItems.java

```diff
@@ -196,6 +198,9 @@ public ApiItems( FeatureFactory registry )
 				.features( AEFeature.STORAGE_CELLS, AEFeature.CREATIVE )
 				.build();
 		this.viewCell = registry.item( "view_cell", ItemViewCell::new ).features( AEFeature.VIEW_CELL ).build();
+		this.cellReadOnly = registry.item( "storage_cell_read_only", ItemReadOnlyStorageCell::new )
+				.features( AEFeature.STORAGE_CELLS, AEFeature.CREATIVE)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) ')' is not preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)